Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/Bundle.properties

```diff
@@ -298,3 +298,38 @@ AutoIngestCasePanel.bnOpen.text=&Open
 AutoIngestCasePanel.bnShowLog.toolTipText=Display case log file for selected case
 AutoIngestCasePanel.bnShowLog.text=&Show Log
 AutoIngestCasePanel.rbGroupLabel.text=Show cases accessed in the last 10:
+AutoIngestControlPanel.tbStatusMessage.text=
+AutoIngestControlPanel.bnShowCaseLog.toolTipText=Display case log file for selected case
+AutoIngestControlPanel.bnShowCaseLog.text=Show Case &Log
+AutoIngestControlPanel.bnDeleteCase.toolTipText=Delete the selected Case in its entirety
+AutoIngestControlPanel.bnDeleteCase.text=&Delete Case
+AutoIngestControlPanel.bnCancelJob.toolTipText=Cancel processing of the current Job and move on to the next Job. This functionality is only available for jobs running on current AIM node.
+AutoIngestControlPanel.bnCancelJob.text=&Cancel Job
+AutoIngestControlPanel.completedTable.toolTipText=The Completed table shows all Jobs that have been processed already
+AutoIngestControlPanel.runningTable.toolTipText=The Running table displays the currently running Job and information about it
+AutoIngestControlPanel.pendingTable.toolTipText=The Pending table displays the order upcoming Jobs will be processed with the top of the list first
+AutoIngestControlPanel.bnPrioritizeCase.toolTipText=Move all images associated with a case to top of Pending queue.
+AutoIngestControlPanel.bnPrioritizeCase.text=Prioriti&ze Case
+AutoIngestControlPanel.bnPause.toolTipText=Suspend processing of Pending Jobs
+AutoIngestControlPanel.bnPause.text=Pause
+AutoIngestControlPanel.bnShowProgress.toolTipText=Show the progress of the currently running Job. This functionality is only available for jobs running on current AIM node.
+AutoIngestControlPanel.bnShowProgress.text=Ingest Progress
+AutoIngestControlPanel.bnOptions.toolTipText=Display options panel. All processing must be paused to open the options panel.
+AutoIngestControlPanel.bnOptions.text=&Options
+AutoIngestControlPanel.bnExit.toolTipText=Exit Application
```

Comment:
I didn't put those in, but from my Visual C++ days, I believe the ampersand just underlines the character that follows and makes the corresponding keyboard key a shortcut to select the object. So, you can press 'O' to select the "Options" button.